docs/book/src/developer/architecture/controllers/cluster-topology.md

```diff
@@ -0,0 +1,18 @@
+# `ClusterTopology` Controller
+
+A `ClusterTopology` controller reconciles the managed topology of a Cluster. 
+
+![Cluster Topology Controller Activity Diagram](../../../images/cluster-topology-controller.png)
+
+Its main responsibilities are:
+1. Extend the Cluster object to use ClusterClass for creating managed topologies. 
+2. Create/Update/Delete managed topologies (this includes continuous reconciliation of the topology managed resources)o.
+3. Introduce mechanisms to allow Cluster-specific customizations of a ClusterClass
+
+The high level workflow of ClusterTopology controller is shown below.
+
+![ClusterTopology Reconciller Component Diagram](../../../images/cluster-topology-reconciller.png)
+
+### Implementations
```

Comment:
```suggestion
### Implementation
```
nit

Reply:
Or maybe: "Additional information" ?

Reply:
I think "additional information" is better, and we should try to move past the link to the proposal and put the implementation details here once ClusterClass is out of alpha.